packages/injected-connector/src/declarations.ts

```diff
@@ -1,8 +1,19 @@
 interface Ethereum {
-  send: unknown
+  request: (args: RequestArguments) => Promise<unknown>
```

Comment:
According to https://docs.metamask.io/guide/ethereum-provider.html#methods , request always returns a promise